### DISCUSSION TITLE
make error: dbcsignaleditor.cpp indicates textactivated is not a member of QCombobox

### DISCUSSION THREAD

**Discussion by sid-ram** [collin80/SavvyCAN#411] [Q&A]
Got this error when making: 
dbc/dbcsignaleditor.cpp: In constructor ‘DBCSignalEditor::DBCSignalEditor(QWidget*)’:
dbc/dbcsignaleditor.cpp:271:48: error: ‘textActivated’ is not a member of ‘QComboBox’
  271 |     connect(ui->cbMultiplexParent, &QComboBox::textActivated(),
      |                                                ^~~~~~~~~~~~~

From dbcsignaleditor.cpp: 
connect(ui->cbMultiplexParent, &QComboBox::textActivated,
            [=]()
            {
                if (currentSignal == nullptr) return;
                if (inhibitMsgProc) return;
                //qDebug() << "Curr text: :" << ui->cbMultiplexParent->currentText();
                //try to look up the signal that we're set to now, remove this signal from existing children list
                //add it to this one, update this signal's parent multiplexor

Appreciate the help. 
Thanks

**Comment by collin80** [comment]
Quite often I forget to update the documentation. textActivated was added in QT 5.14 so my guess is you're trying to use an older version of QT. Yes, the documentation says you can use QT 5.13 but it's not true. In general, you should be using the newest stable version of QT. However, do not try to use QT 6 yet. So far as I know they have not yet added support for all of the various plugins QT is supposed to come with. One of the big omissions so far is lack of CAN bus support in QT6.

**Reply by sid-ram** [reply]
Thanks Collin. I had QT6 and now changed it to QT5.14 and got it working.
